app/controllers/provider_interface/offers_controller.rb

```diff
@@ -110,6 +110,13 @@ def query_service
       )
     end
 
+    def persist_conditions_service(conditions)
```

Comment:
Is there a reason why this method isn't `update_offer_conditions_service` or `update_offer_conditions` ? It's easy enough to understand as is but seems superfluous to alias this to `persist_...`, I may not have sufficient context.

Reply:
Yeah that's a good point. I can probably switch to `update_offer_...` throughout over `persist_...`